packages/grid/test/keyboard-navigation-cell-button.test.js

```diff
@@ -0,0 +1,75 @@
+import { expect } from '@esm-bundle/chai';
+import { arrowLeft, arrowRight, aTimeout, fixtureSync } from '@vaadin/testing-helpers';
+import sinon from 'sinon';
+import '../vaadin-grid.js';
+import { flushGrid } from './helpers.js';
+
+let grid;
+
+function getRowCell(rowIndex, cellIndex) {
+  return grid.$.items.children[rowIndex].children[cellIndex];
+}
+
+function getRowFirstCell(rowIndex) {
+  return getRowCell(rowIndex, 0);
+}
+
+describe('keyboard navigation - cell button', () => {
+  let cell;
+
+  beforeEach(async () => {
+    grid = fixtureSync(`
+      <vaadin-grid>
+        <vaadin-grid-column path="name" _cell-button></vaadin-grid-column>
+        <vaadin-grid-column path="name"></vaadin-grid-column>
+      </vaadin-grid>
+    `);
+
+    grid.items = [{ name: 'foo' }, { name: 'bar' }];
+
+    grid._observer.flush();
+    flushGrid(grid);
+
+    await aTimeout(0);
+
+    cell = getRowFirstCell(0);
+  });
+
+  it('should create a focusable div with role="button" inside the cell', () => {
+    expect(cell.firstChild.localName).to.equal('div');
+    expect(cell.firstChild.getAttribute('role')).to.equal('button');
+  });
+
+  it('should set tabindex on the focusable div inside the cell', () => {
+    expect(cell.hasAttribute('tabindex')).to.be.false;
+    expect(cell.firstChild.getAttribute('tabindex')).to.equal('0');
+  });
```

Comment:
Does it make sense to test the reverse, that it does not render a div with role and tabindex if the mode is disabled?